src/main/java/nl/igorski/lib/audio/MWEngine.java

```diff
@@ -308,6 +308,7 @@ public void run()
     {
         //DebugTool.log( "MWEngine STARTING NATIVE AUDIO RENDER THREAD" );
 
+        android.os.Process.setThreadPriority( Process.THREAD_PRIORITY_URGENT_AUDIO );
         handleThreadStartTimeout();
```

Comment:
Improvement suggested by @mattlogan to maintain audio stability for application suspension. This should also be beneficial in general usage (ensures MWEngine processing has the highest Linux thread scheduling priority).